### DISCUSSION TITLE
Humble Bazooka PCE

### DISCUSSION THREAD

**Discussion by haydnkilyan** [darthcloud/BlueRetro#629] [Q&A]
This device is awesome, however, I'm having some issues with this combination...... 8Bitdo M30, 6 Button mode, mutiltap on output 1, using a Terraoinion Super HD System3 Pro..... when in the menu all works as expected. I have mapped the buttons and when in pad test software all is fine.... however when I load a game (e.g Bomberman '93) it is like the direction buttons are held down (cycles through the menu options automatically- without any buttons being pressed) I am now using and SN30 Pro in 2 button mode and it works fine. Lots of variables to test here, wondered if anyone had experienced something similar?

**Comment by DoomyDoomer** [comment]
On the original PC Engine, many 2 (and 3) button games were not compatible with 6 button controllers. I think what you've described is the adapter acting as the original hardware would. For example, Bomberman '93 I believe is a 2 button only game and glitches with 6 button mode. 3 button controllers actually acted the same as a 2 button controller but with the ability to map the third button to run or select. 6 button controllers functioned entirely differently and there's only a handful of games that truly support them.

**Reply by haydnkilyan** [reply]
That is really interesting- I have and avenue 6 button controller, but have only ever used with Street Fighter 2, I'll give it a test tonight

**Reply by darthcloud** [reply]
The blueretro setting is the equivalent of the avenue 6 mode switch (A, B setting)

If you put the avenue 6 in 6 buttons mode in a 2 buttons games it wont work.

**Reply by haydnkilyan** [reply]
So if I use an 8bitdo m30 as my main controller, and map the 6 buttons accordingly for a game like street fighter 2, there is no way to use it with a 2 button game without diving back into the settings?

**Reply by darthcloud** [reply]
Exact

**Reply by haydnkilyan** [reply]
Thanks for clearing that up ☺️ I wonder if there is a way to map the A/B switch
